crypto/s2n_hkdf.c

```diff
@@ -126,7 +125,7 @@ static int s2n_libcrypto_hkdf_extract(struct s2n_hmac_state *hmac, s2n_hmac_algo
     const EVP_MD *digest = NULL;
     POSIX_GUARD_RESULT(s2n_hmac_md_from_alg(alg, &digest));
 
-    size_t pseudo_rand_key_len = (size_t) pseudo_rand_key->size;
+    size_t pseudo_rand_key_len = 0;
```

Comment:
Why was this a problem? At worst, I'd expect the existing value to be ignored when it's set.

Reply:
It's not a problem, I just thought it would make it more clear what HKDF_extract is doing with that argument when reading the code. I also updated the variable name to make it more clear. Is that reasonable, or does it make sense to keep it set to the correct size?